src/platform/site-wide/component-library-analytics-setup.js

```diff
@@ -32,6 +32,29 @@ const analyticsEvents = {
   TextInput: [{ action: 'blur', event: 'int-text-input-blur' }],
   'va-checkbox': [{ action: 'change', event: 'int-checkbox-option-click' }],
   'va-text-input': [{ action: 'blur', event: 'int-text-input-blur' }],
+  'va-accordion': [
+    { action: 'expand', event: 'int-accordion-expand' },
+    { action: 'collapse', event: 'int-accordion-collapse' },
+  ],
+  'va-additional-info': [
+    { action: 'expand', event: 'int-additional-info-expand' },
+    { action: 'collapse', event: 'int-additional-info-collapse' },
+  ],
+  'va-alert': [{ action: 'linkClick', event: 'nav-alert-box-link-click' }],
+  'va-breadcrumbs': [
+    { action: 'linkClick', event: 'nav-breadcrumb-link-click' },
+  ],
+  'va-checkbox-group': [
+    { action: 'change', event: 'int-checkbox-group-option-click' },
+  ],
+  'va-loading-indicator': [
+    { action: 'displayed', event: 'loading-indicator-displayed' },
+  ],
+  'va-radio': [{ action: 'change', event: 'int-radio-button-option-click' }],
+  'va-select': [{ action: 'change', event: 'int-select-box-option-click' }],
+  'va-progress-bar': [{ action: 'change', event: '' }],
```

Comment:
@bsmartin-ep do you happen to know what the events would be for: Progress Bar, Segmented Progress Bar and Telephone? I couldn't find them in the Component Library and they weren't listed above was wondering if they had been created previously